Files/ViewModels/ItemViewModel.cs

```diff
@@ -1586,6 +1582,7 @@ public async Task<int> EnumerateItemsFromStandardFolderAsync(string path, Type s
                 }
                 else if (hFile.ToInt64() == -1)
                 {
+                    rootFolder ??= await FilesystemTasks.Wrap(() => StorageFileExtensions.DangerousGetFolderFromPathAsync(path));
```

Comment:
`StorageHelpers.ToStorageItem<>()`?

Reply:
Mm I wouldn't call ToStorageItem from inside ItemViewModel as it ends up calling methods inside ItemViewModel itself. Also it calls GetRootFromPathAsync which I don't want in this case.